pkg/config/dn.go

```diff
@@ -0,0 +1,82 @@
+// Copyright 2023 LY Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
```

Comment:
maybe in util directory?

Reply:
Packages named `util` should be avoided. As it is only used in `config` pkg now, prefer to keep in config pkg.

> Avoid meaningless package names. Packages named util, common, or misc provide clients with no sense of what the package contains. This makes it harder for clients to use the package and makes it harder for maintainers to keep the package focused.

source: https://go.dev/blog/package-names

Reply:
okie